app/components/github_integration/webhooks/utils.py

```diff
@@ -55,12 +55,17 @@ class EmbedContent(NamedTuple):
     title: str
     url: str
     body: str | None = None
+    description: str | None = None
 
     @property
     def dict(self) -> EmbedContentArgs:
         args: EmbedContentArgs = {"title": self.title, "url": self.url}
         if self.body:
             args["description"] = truncate(self.body, 500)
+        if self.description:
+            # If provided a description explicitly, don't truncate. However, discord has
+            # a description character size limit.
+            args["description"] = truncate(self.description, 6000)
```

Comment:
The diff is horrible but I essentially moved the description if above the body if and made the body if an elif.

```suggestion
        if self.description:
            # If provided a description explicitly, don't truncate. However, discord has
            # a description character size limit.
            args["description"] = truncate(self.description, 6000)
        elif self.body:
            args["description"] = truncate(self.body, 500)
```